upkie/cpp/actuation/BulletInterface.cpp

```diff
@@ -353,6 +361,33 @@ void BulletInterface::read_joint_sensors() {
     result.torque = sensor_state.m_jointMotorTorque;
   }
 }
+void BulletInterface::get_nominal_masses() {
+  const int nb_links = bullet_.getNumJoints(robot_);
+  b3DynamicsInfo info;
+  for (int link_id = 0; link_id < nb_links; ++link_id) {
+    bullet_.getDynamicsInfo(robot_, link_id, &info);
+    nominal_masses[link_id] = info.m_mass;
+    for (int i = 0; i < 3; ++i) {
```

Comment:
The compiler will probably unroll this loop when we compile in release. (For three items I'd probably have unrolled it by hand :blush: but both are fine.)